ssl/ssl_lib.c

```diff
@@ -4343,6 +4343,80 @@ const CTLOG_STORE *SSL_CTX_get0_ctlog_store(const SSL_CTX *ctx)
     return ctx->ctlog_store;
 }
 
+void
+SSL_CTX_set_early_cb(SSL_CTX *c, ssl_early_cb_fn cb, void *arg)
```

Comment:
Please put the return  type on the same line as the rest of the declaration. Same for other funcs below.